node/src/braid/mod.rs

```diff
@@ -1,11 +1,13 @@
 use crate::bead::Bead;
-use crate::utils::BeadHash;
-use ::serde::Serialize;
-use std::collections::HashSet;
+use crate::utils::{retrieve_bead, BeadHash};
+use serde::Serialize;
+use std::cmp::Ordering;
+use std::collections::VecDeque;
+use std::collections::{HashMap, HashSet};
+pub const FIXED_BEAD_WORK: u32 = 1;
```

Comment:
"work" is a 256-bit quantity. It's an estimate of the number of sha256d hashes performed and is 1/target.

This misunderstanding appears several times in this file...